src/mms_client/types/enums.py

```diff
@@ -19,8 +19,55 @@ class AreaCode(Enum):
     OKINAWA = "10"
 
 
+class Direction(Enum):
+    """Represents the reserve direction of the offer."""
+
+    SELL = "1"  # Increasing the reserves (sell)
+    # Note: Support for the BUY direction was removed from the MMS API
```

Comment:
How are we supposed to "Buy low, sell high" if we aren't allowed to buy? smh my head.

Reply:
You sell hard enough that it overflows into a negative number and end up buying.